WordSearch.java

```diff
@@ -0,0 +1,113 @@
+import java.util.*;
+
+/**
+ * Created by Alice Colt.
+ */
+public class WordSearch {
+
+
+    // Search for a WORD - using list
+
+    static List<String> wordSearchGrid(char[][] characters,
+                                       int row,
+                                       int column,
+                                       Dictionary dictionary){
```

Comment:
"row" and "column" can be inferred from the size of "characters". Either do that inferring, or if you really want them as parameters I suggest a consistency check.